torchgen/gen.py

```diff
@@ -481,7 +482,10 @@ class RegisterSchema:
     def __call__(self, f: NativeFunction) -> Optional[str]:
         if not self.selector.is_native_function_selected(f):
             return None
-        return f"m.def({cpp_string(str(f.func))});\n"
+        if len(f.tags) == 0:
```

Comment:
snippet from `RegisterSchema.cpp`:
```
  m.def("rename_(Tensor(a!) self, Dimname[]? names) -> Tensor(a!)", std::unordered_set<Tags>({inplace_view}));
  m.def("rename(Tensor(a) self, Dimname[]? names) -> Tensor(a)");
  m.def("align_to(Tensor(a) self, Dimname[] names) -> Tensor(a)");
```

Reply:
This is a separate issue... but we probably dont want the tags to specify information that are already present in the schema, e.g. `inplace_view`

Reply:
cc. @bdhirsh

Reply:
`inplace_view` is actually pretty useful because you can't tell just from the aliasing info `(a!)` if the op mutates the tensor's data vs. metadata.

We can technically figure it out by noticing that `transpose(Tensor(a), int dim0, int dim1) -> Tensor(a)` and `transpose_(Tensor(a!), int dim0, int dim1) -> Tensor(a!)` have similar schemas that can be paired up (so the mutable variant of a view op is an "inplace view"). But the tag is actually pretty useful in the codegen- we can use it for error checking, and once it's set then we don't need the grouping info anymore to know if we have an inplace view.